PageLayout.Content should not use main landmark by default. (#3154)
primer/react: .changeset/eleven-humans-sneeze.md

---
"@primer/react": patch
---

> PageLayout.Content no longer renders as `main` by default. Instead, developers must specify this using the `as` prop, which defaults to `div`.

Reviewer: ```suggestion
PageLayout.Content no longer renders as `main` by default. Instead, developers may add a `main` landmark within `Pagelayout.Content` themselves.
```
